zuul.d/trigger_jobs.yaml

```diff
@@ -0,0 +1,55 @@
+---
+- job:
+    name: downstream-va-hci-trigger-job
+    parent: create-nested-virt-trigger-from-github-pr
+    irrelevant-files: &ir_files
+      - .*/*.md
+      - ^.github/.*$
+      - ^LICENSE$
+      - ^OWNERS$
+      - ^OWNERS_ALIASES$
+      - ^PROJECT$
+      - ^README.md$
+      - ^renovate.json$
+      - ^kuttl-test.yaml$
+      - molecule/.*
+      - molecule-requirements.txt
+      - .github/workflows
+      - docs/.*
+      - contribute/.*
+      - roles/.*/molecule/.*
+      - ci/playbooks/pre-commit.yml
+      - ci/playbooks/pre-doc.yml
+      - ci/playbooks/run-doc.yml
+      - ci/playbooks/molecule-prepare.yml
+      - ci/playbooks/molecule-test.yml
+      # ci-framework
+      - .ansible-lint
+      - .config/molecule/.*
+      - .pre-commit-config.yaml
+      - .readthedocs.yaml
+      - .spellcheck.yml
+      - roles/dlrn_report
+      - roles/dlrn_promote
+      - roles/devscripts
```

Comment:
I think we want to trigger va-hci when `devscripts`, `dnsmasq`, `reproducer`and `virtualbmc` roles are affected: they are all used to get the infra layer, OCP and some services.

Reply:
as for nat64_appliance, it might be requested if we get va-hci with IPv6

Reply:
`libvirt_manager`?

Reply:
Done

Reply:
OK, didn't see you listed the "files" + "irrelevant-files" - iirc this doesn't work like that afaik, because you list the same path in both blocks.

And there are far more roles to track down, lemme list the ones that should trigger.